doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
 Chapel Quickstart Instructions
 ==============================
 
-These instructions are designed to help you get started with
-a source distribution of the latest Chapel release.
+These instructions are designed to help users get started with a
+single-locale (shared-memory) implementation of Chapel from the source
+distribution of the release as quickly as possible.  If you are
+interested in taking the time to understand Chapel's configuration
+options, build process, and installation more thoroughly, please refer
+to :ref:`readme-chplenv` and :ref:`readme-building` instead.
 
-In the following instructions, note that building and using Chapel as
-described in steps 2-5 disables optional and advanced features in the
-interest of getting you a clean build as quickly as possible. Later
-sections explain how to re-build in the preferred configuration and how to
-enable more features, such as distributed memory execution.
+These instructions first have you build a minimal, stripped-down
+version of Chapel to reduce build times and the potential for
+third-party portability issues.  Once you are interested in a
+full-featured version of Chapel, refer to
+:ref:`using-a-more-full-featured-chapel` below.
 
 
-0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system
-   tools and packages you may need to have installed to build and run Chapel.
+0) See :ref:`readme-prereqs` for information about system tools and
+   packages you should have available to build and run Chapel.
 
-1) If you don't already have Chapel 1.25, see
-   https://chapel-lang.org/download.html .
 
-2) If you are using a source release, build Chapel in a *quickstart*
-   configuration. The *quickstart* script used below is designed to help you
-   get started with Chapel. After that, you may want to switch to a more
-   full-featured configuration. See using-a-more-full-featured-chapel_ below.
+1) If you don't already have the Chapel 1.25 source release, see
```

Comment:
Should we drop the version name here to make the document more future-proof? Similarly for the filenames below

Reply:
This is admittedly a maintenance issue, but we haven't had a problem with it falling out-of-date, and should have even less of a problem as Max automates things like this.  I think the obvious benefit of naming the version number is that, for some who read this and seem very UNIX-/tech-unsavvy, putting something like `cd chapel-[your-version-number]` would cause them to type that in and then wonder why it doesn't work.  So we've traditionally decided to take the maintenance hit here, and I'd like to advocate that we continue to do so.